analysers/analyser_osmosis_wikipedia.py

```diff
@@ -26,44 +26,34 @@
 sql10 = """
 SELECT
   (array_agg(tid))[1:10],
-  ST_AsText(any_locate((array_agg(type))[1], (array_agg(id))[1])),
+  ST_AsText(any_locate(substring(MIN(tid), 1, 1), substring(MIN(tid), 2)::bigint)),
```

Comment:
It is not the min as expected, because it compare on strings.

Reply:
Ah, true, n56789 would be after n100000. 
However, my goal was just to have a stable output (as long as no object is added/removed) to get rid of the jumping markers, so whether it's alphabetically the first or numerically the first should not make a difference, right?

Reply:
I think older is more stable.
It just about concatenate type later.

Reply:
Not sure how to get the oldest element.
If I add a new way today, the ID will be [around 1185000000](https://www.openstreetmap.org/way/1185000000). A [node with the same ID](https://www.openstreetmap.org/node/1185000000) would be 12 years old. So comparison by id will not retrieve the oldest element

Reply:
Just order by type and id will do the job. The older by type is sufficient, isn't it ?

Reply:
I'll have another look at a later moment.

Just calling ORDER BY won't work, since I can't add id or type to GROUP BY or it'll be filtered out. Probably have to modify the COUNT

Reply:
Nope, also with `COUNT(w)` it doesn't work. Probably I misunderstood what you meant @frodrigo, where would I have to call the ORDER BY type, id?

Reply:
Actually, the issue ID changes anyway if elements are added or deleted, right? So is it really needed to be the same element every time, also when elements are added? (Of course, the jumping when there's no changes must be fixed)